overlord/ifacestate/helpers.go

```diff
@@ -331,6 +331,37 @@ func (m *InterfaceManager) reloadConnections(snapName string) ([]string, error)
 	return result, nil
 }
 
+func (m *InterfaceManager) refreshStaticConnectionAttrs(snapInfo *snap.Info) error {
```

Comment:
Can we unit test this somehow? Maybe something basic/trivial?

Reply:
Yes, Pawel wrote unit tests yesterday. I will incorporate them into this PR now.